websites/A/AnimeFLV/dist/metadata.json

```
@@ -15,8 +15,8 @@
     "zh_CN": "AnimeFLV是一个视频网站，您可以在其中观看带有西班牙语字幕的动漫。"
   },
   "url": [
-    "animeflv.ru",
-    "animeflv.net"
+    "animeflv.net",
+    "animeflv.ru"
   ],
   "version": "1.1.6",
```

```suggestion
  "version": "1.1.7",
```
if you do not change the version, the system will not think that you have changed file/source code, which means that your merge will not be applied